### ISSUE TITLE
Incorrect equation for the Hamiltonian in Max Cut

### ISSUE THREAD

**Issue by camitava0321** [Qiskit/qiskit#1047]
## Incorrect equation for the Hamiltonian in Max Cut
- While going through MaxCut documentation, I noticed a possible mistake while formulating C(Z) from C(X) - if x-> (1-z)/2 then (1-x) should be just z/2, not (1+z). If this is so then the Hamiltonian will eventually be different. Also after C(Z) the const term is being rendered as latex code.
- https://qiskit.org/documentation/tutorials/optimization/6_examples_max_cut_and_tsp.html
![image](https://user-images.githubusercontent.com/25051480/93669671-693e4500-fab3-11ea-9a12-0a2b0fd20f91.png)

**Comment by Claeb101** [comment]
I'm fairly new to quantum computing but would like to contribute where I can. This seems like a good place to start. Can I work on this issue?

**Comment by mikeumus** [comment]
Hi @Claeb101, glad you're interested. However, I'm just an observer of the Qiskit repo. 
You can reference the [README.md](https://github.com/Qiskit/qiskit/blob/master/README.md) and [CONTRIBUTING.md](https://qiskit.org/documentation/contributing_to_qiskit.html)

@camitava0321, any advice for Claeb101 for starting out on this ticket?

**Comment by Claeb101** [comment]
Oh I'm so sorry. I wasn't quite sure who to ask if I could contribute. I will ask @camitava0321 in the future. I take it I'm allowed to work right away on any issues then instead of asking every time?

**Comment by mikeumus** [comment]
@Claeb101 yeah I'm sure if your PRs match the requirements in `CONTRIBUTING.md` that they'd be welcome. 😃👍
